src/app/components/meta/meta.component.html

```diff
@@ -33,7 +33,9 @@
     | {{ video.width }} x {{ video.height }}
 
     | {{ video.fileSizeDisplay }}
-
+    
+    | {{ video.bitrate + ' mb/s' }}
```

Comment:
I'm happy to accept as is. But currently if `bitrate` property is absent (as it would be for old Hubs), there would be an empty display. I recommend something like the line below, for example:
`{{ video.bitrate ? '| ' + video.bitrate + ' mb/s' : '' }}`

Reply:
There will always be a bitrate property, because it's calculated from videoSize and duration, which are present in every video there is (if I understand correctly).

Reply:
Oh you're right! I'm sorry, in my mind, before I saw your code, I was thinking you were grabbing the bitrate from what `FFmpeg` was returning (when initially scanning the video), and then I commented based on my guess about your code rather than your code 😓 

I like this implementation rather than what I was thinking, since it provides previously-scanned videos with a bitrate estimate!

I'll merge this in soon 👏